src/feedback/vue/submitFeedback.vue

```diff
@@ -60,6 +60,7 @@ import { defineComponent } from 'vue'
 import { WebviewClientFactory } from '../../webviews/client'
 import saveData from '../../webviews/mixins/saveData'
 import { FeedbackWebview } from './submitFeedback'
+import { transformByQState } from '../../codewhisperer/models/model'
```

Comment:
I just noticed this is in a `.vue` (frontend) file. That is probably why the webpack build was failing.